src/bin/inx-chronicle/main.rs

```diff
@@ -131,7 +130,8 @@ async fn main() -> eyre::Result<()> {
 
     // Allow the user to abort if the tasks aren't shutting down quickly.
     tokio::select! {
-        _ = process::interrupt_or_terminate() => {
+        res = process::interrupt_or_terminate() => {
+            res?;
```

Comment:
Well, that's a good point I can absolutely agree with. In my defense tho ... that is a functional change in behavior, because before ... we just panicked.